### PR TITLE
V0.0.2 display fee est

### PR DESCRIPTION
**Pull request by matt-auckland**
Follow up to https://github.com/NAVCoin/NavMorph/pull/216

### CODE REVIEW

File: frontend/src/app/pages/status/status.component.ts [hunk 1]
```diff
@@ -36,7 +36,8 @@ export class StatusPage implements OnInit {
   constructor(
     private OrderService: OrderService,
     private GenericFuncs: GenericFunctionsService,
-    private router: Router
+    private router: Router,
+    private dataService: SendPageDataService
```
Comment (red010b37) [inline frontend/src/app/pages/status/status.component.ts]: your service vars are not consistent - mix of upper a lower case.

also for private vars I prefer the _ notations 


e.g
 private OrderService: OrderService,

would be
 private _orderService: OrderService,

**Review by red010b37 — COMMENTED** [review]
Left a note for you around consistency or the DI vars

File: frontend/src/app/pages/status/status.component.ts (hunk 1 above)
Comment (matt-auckland) [reply frontend/src/app/pages/status/status.component.ts]: _ notation seems nice. I'll switch to it here. Won't be overnight but, I'll eventually propagate this notation style around the rest of the front end as well.

Comment (red010b37) [reply frontend/src/app/pages/status/status.component.ts]: Yea thats totally cool!

Comment (zanuka) [reply frontend/src/app/pages/status/status.component.ts]: roger that

**Pull request closed without merging**